project-fortis-pipeline/ops/storage-ddls/cassandra-setup.cql

```diff
@@ -1,75 +1,82 @@
 DROP KEYSPACE IF EXISTS fortis;
 CREATE KEYSPACE fortis WITH replication = {
     'class': 'SimpleStrategy',
-    'replication_factor': 3
+    'replication_factor': 1 // TODO: change to 3 before merging
```

Comment:
Reminder to not check this in.

Note that the run-node.sh file should already be adapting the replication factor to 1 since the appropriate environment variable is being set in docker-compose.yml

https://github.com/CatalystCode/project-fortis/blob/0f2d35181104548e1e46ff0f0d1c0164623de2c5/docker-compose.yml#L42

https://github.com/CatalystCode/project-fortis/blob/0f2d35181104548e1e46ff0f0d1c0164623de2c5/project-fortis-services/docker/run-node.sh#L30-L35